drivers/eeprom/eeprom_mchp_xec.c

```diff
@@ -321,6 +365,12 @@ static const struct eeprom_driver_api eeprom_xec_api = {
 	.size = eeprom_xec_size,
 };
 
+#ifdef CONFIG_PM_DEVICE
+#define XEC_EEPROM_PM_DEV_DEFINE(i) PM_DEVICE_DT_INST_DEFINE(i, eeprom_xec_pm_action);
+#else
+#define XEC_EEPROM_PM_DEV_DEFINE(i) /* Not used */
+#endif
```

Comment:
`PM_DEVICE_DT_INST_DEFINE()` does not need to be wrapped in `fdef CONFIG_PM_DEVICE` as it is already a no-op if `CONFIG_PM_DEVICE` is not enabled.

Reply:
Updated the code.